sktime/forecasting/callbacks/mlflow.py

```diff
@@ -0,0 +1,242 @@
+"""MLFLow callback for logging metrics and plots to MLFlow."""
+from typing import Optional
+
+import mlflow
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
```

Comment:
Same. Plus, I don't think we have `plotly` as a dependency at this point, not directly at least.

Reply:
Changed import

Reply:
Thanks. But I am a bit confused where exactly `plotly` is being installed. It's not in `pyproject.toml`, will it be available as it'll be installed as dependency of some other libraries?